### ISSUE TITLE
[Community Bounty] Consumable Glossary - Augur.Guide

### ISSUE THREAD

**Issue by msagansk** [AugurProject/augur#1236]
Original idea outline: "Take the Augur Glossary (in the Augur Docs) and create some sort of crash course / means of digesting and consuming the terms, with clean deep-linking to important concepts. You should create something where anyone who wants to learn more in depth about Augur can easily consume the details surrounding forks, reporting, trading, different states of Augur, etc. Think of example websites made by RadarRelay for example: WETH.io, Gwei.io, EthGas.io, etc."

I've started https://Augur.Guide which is an open source Github Pages site under the MIT License. My vision for the site is for it to become the go-to source of information for regular Augur users. This differs from https://docs.augur.net which has developers (and power users) as its main target audience. I'd like Augur experts to be able to answer beginner questions by just being able to send a deep-link to Augur.Guide.

By having Augur.Guide be open source, it gives the site and content more staying power. If I happen to disappear sometime in the future for whatever reason, other people can pick up the torch and keep the content up to date. We all know that Augur will continue to improve in the future with contract upgrades and UI updates, so keeping Augur.Guide updated will be critical to maintain usefulness and prevent confusion.

I've already created the beginnings of the site with a UI layout, navigation structure, and a few pages of content. I will be limiting the scope of the site to just v1 of the Augur contracts for now, with perhaps small notes here and there about how certain aspects will change in v2.

My plan is somewhat fluid and open to suggestions for change, but for right now the basic structure of the site will be:

General information applicable to everyone:

1. Introductory/Getting Started information: What is Augur, how to access it, how it works.
2. Life of a market: A process flow chart about the different states a market goes through. I'm currently deliberating on whether I should include information about the forking process near the beginning of the site or pushing it to its own separate page later on. Right now I'm favoring the idea of presenting the best/easy case first (no disputes), then showing how it works when a dispute takes place, all the way to a fork.
3. Market Types: Yes/No, Categorical, Scalar explained

Then I start breaking up the information depending on the type of user:

4. Market Creators: How to create a market, an explanation of all the parameters and bonds required. Tips and a check sheet on how to avoid making an invalid market. Templates/examples of properly crafted markets that can be duplicated by just adjusting a couple variables.
5. Traders: How trading works, watching out for markets likely to be invalid, trading SHARES vs. ETH, how the fees work, waiting for settlement vs. selling SHARES at an early discount
6. Reporters: A reiteration of the dispute process, commonly accepted rules for markets being invalid (and why they exist). Notes on potential edge cases for markets. Participation tokens.
7. Developers: I may cut this out, depending on how much content I feel I can provide here. I may just end up linking to docs.augur.net on the External resources page instead. Off the top of my head I know I can provide some additional insight such as the contract addresses deployed on mainnet.

Wrapping it up with:

8. External resources - a curated list of 3rd party tools and applications that built something on top of Augur.
9. Glossary - Similar to the glossary on docs.augur.net but focused on regular users. The other pages on the site will have deep-links to terms present on this page.

I plan to chip away at these pages over the coming 4 weeks with a planned finish date of April 1, 2019. Over this time I welcome any and all feedback to help improve the site. 

My best guess for the first 50% milestone will be when pages on steps 1-5 (from above) are written, which I will aim to complete by March 15, 2019. Steps 6-9 will then follow for the final 50%.

I do intend to keep improving the site after the bounty as well, but it will be done on a spare time basis. I do thank you for the opportunity at the bounty, as it provides me with justification to sprint towards enough progress where Augur.Guide can be regularly shared with the community.

My ETH Address: 0xf733454BB020B76cFc9c4Ad80367BAb412EdBFfb

**Comment by Chwy5** [comment]
Its a good idea and something that should be explored, but the timing is not right for it as there will be many changes including usability and terminology.  Creating this now would be a waste of time.  Needs to be done after the redesign

**Comment by msagansk** [comment]
What are you specifically referring to when you say "it" Chwy5? Some aspect in particular or this whole bounty in general?

**Comment by Chwy5** [comment]
General Information should remain the same, 
Any UI layout, navigation structures for market creation, trading, and reporting/disputing will change at some point, any how to guide, on specifics to trading, will be different, Glossary will have more terms not currently used

**Comment by msagansk** [comment]
I agree. I have specifically avoided putting in any UI layout or specific steps into the site as it will become obsolete very soon. And yes, the site will have to be updated once v2 comes out. But there is already quite a bit of informative value written on the site as defined by the agreed upon bounty, and most of it will still be relevant when v2 is released. The site is already in a state where it can be useful to others, right now.

In fact, I believe I have completed the requirements for 50% of the bounty. Steps 1-5 outlined above have been completed. 

Regarding 2., I broke out the "Life of a Market" into its simplest case first, with a separate page built under the "Reporting" section for a more complex example involving disputes. Further still, the forking process was also broken out into its own page. Regarding 7., I've cut it out for now, but may add it back in if I feel like I have more to add than docs.augur.net and I have the time. I'd rather focus the second 50% on building a great Glossary.

The second 50% will mainly revolve around writing the reporters pages that are still empty or incomplete, more detail on the external resources page, and of course - the glossary. I will then circle back throughout the site, linking key terms used on the various explanatory pages to their glossary definition. As I circle back, I will refine and flush out any parts of the site to make everything a more cohesive whole.

Feedback is appreciated.

**Comment by Chwy5** [comment]
The Augur.Guide looks good.  Only a few small tweaks will be needed after the redesign is released and some additions to the glossary.

**Comment by msagansk** [comment]
Ok, I've finished parts 6-9 as outlined above - so I believe I have completed the bounty! I did end up cutting out the developer section (docs.augur.net is great), absorbing some of the developer resources into the "External Resources page" instead.

For the glossary page, I built off of the one from docs.augur.net. I took out the developer specific information, updated some definitions, and added in a few of my own.

I think the site is looking pretty good. I've referred a couple people who have asked questions about Augur and it seemed to help. While there is much more that can be done to the site, and it will have to be updated as Augur changes over time, it is in a usable and useful form now.

Of course, I am always open to suggestions for future improvement. I will continue working on the site in my spare time. I had a lot of fun working on it and it helped me understand Augur better as well. Thanks for the opportunity.
